typings/index.d.ts

```diff
@@ -889,7 +889,7 @@ declare module 'discord.js' {
     public syncedAt: number;
     public syncing: boolean;
     public type: string;
-    public user: User;
+    public user: User | null;
```

Comment:
`user` isnt actually set to null, its just left as undefined if the user isn't present

Reply:
ah, so should this remain unchanged? I saw that Guild#premiumSubscriptionCount is typed as `number | null` but the code is like this

Reply:
The property isn't set when `data.user` is missing, however, you can either make the field as optional (see suggestion below), or default it to `null`.
```suggestion
    public user?: User;
```